integration-tests/features/steps/common.py

```diff
@@ -176,16 +176,38 @@ def perform_valid_manifest_post(context, manifest, url):
     print(response.json())
 
 
-@when("I send NPM package manifest {manifest} to stack analysis")
-def npm_manifest_stack_analysis(context, manifest):
+def send_manifest_to_stack_analysis(context, manifest, name, endpoint):
     filename = 'data/{manifest}'.format(manifest=manifest)
-    files = {'manifest[]': ('package.json', open(filename, 'rb'))}
-    endpoint = urljoin(context.coreapi_url, 'api/v1/stack-analyses')
+    files = {'manifest[]': (name, open(filename, 'rb'))}
+    endpoint = urljoin(context.coreapi_url, endpoint)
     response = requests.post(endpoint, files=files)
     response.raise_for_status()
     context.response = response
 
 
+def stack_analysis_endpoint(context, version):
+    endpoint = {"1": "/api/v1/stack-analyses",
+                "2": "/api/v1/stack-analyses-v2"}.get(version)
+    if endpoint is None:
+        raise Exception("Wrong version specified: {v}".format(v=version))
+    return urljoin(context.coreapi_url, endpoint)
```

Comment:
You do `urljoin(context.coreapi_url, endpoint)` in two places, here and in `send_manifest_to_stack_analysis()`